static/js/containers/LearnerSearchPage.js

```diff
@@ -0,0 +1,54 @@
+// @flow
```

Comment:
Our pattern is using a container component whenever we need to connect to the store. Some of our container components are also presentational (i.e. they actually render UI) like `SettingsPage` and `UserMenu`. Here I think there's too much UI being render to put it all in the same component.

We do use container components for single pages quite often - in fact for every page in the app.